posthog/hogql/database/schema/sessions_v1.py

```diff
@@ -78,9 +78,6 @@
     "$entry_referring_domain": StringDatabaseField(name="$entry_referring_domain"),
     "$entry_gclid": StringDatabaseField(name="$entry_gclid"),
     "$entry_gad_source": StringDatabaseField(name="$entry_gad_source"),
-    "$event_count_map": DatabaseField(name="$event_count_map"),
-    "$pageview_count": IntegerDatabaseField(name="$pageview_count"),
-    "$autocapture_count": IntegerDatabaseField(name="$autocapture_count"),
```

Comment:
Looks like this will break 3 SQL insights in the US ([Metabase](https://metabase.prod-us.posthog.dev/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJuYXRpdmUiLCJuYXRpdmUiOnsicXVlcnkiOiJTRUxFQ1QgKiBGUk9NIHBvc3Rob2dfZGFzaGJvYXJkaXRlbSBXSEVSRSBxdWVyeTo6dGV4dCBMSUtFICclJGV2ZW50X2NvdW50X21hcCUnIE9SIHF1ZXJ5Ojp0ZXh0IExJS0UgJyUkcGFnZXZpZXdfY291bnQlJyBPUiBxdWVyeTo6dGV4dCBMSUtFICcmJGF1dG9jYXB0dXJlX2NvdW50JScgIiwidGVtcGxhdGUtdGFncyI6e319LCJkYXRhYmFzZSI6MzR9LCJkaXNwbGF5IjoidGFibGUiLCJwYXJhbWV0ZXJzIjpbXSwidmlzdWFsaXphdGlvbl9zZXR0aW5ncyI6e319)) and 6 in the EU ([Metabase](https://metabase.prod-eu.posthog.dev/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJuYXRpdmUiLCJuYXRpdmUiOnsicXVlcnkiOiJTRUxFQ1QgKiBGUk9NIHBvc3Rob2dfZGFzaGJvYXJkaXRlbSBXSEVSRSBxdWVyeTo6dGV4dCBMSUtFICclJGV2ZW50X2NvdW50X21hcCUnIE9SIHF1ZXJ5Ojp0ZXh0IExJS0UgJyUkcGFnZXZpZXdfY291bnQlJyBPUiBxdWVyeTo6dGV4dCBMSUtFICcmJGF1dG9jYXB0dXJlX2NvdW50JScgIiwidGVtcGxhdGUtdGFncyI6e319LCJkYXRhYmFzZSI6MzR9LCJkaXNwbGF5IjoidGFibGUiLCJwYXJhbWV0ZXJzIjpbXSwidmlzdWFsaXphdGlvbl9zZXR0aW5ncyI6e319)). Is there a straightforward way to migrate them (e.g. with a more manual way to obtain these counts), or not worth it?

Reply:
Hmm. The V2 table is so close to being ready (probably today in EU) that it would make sense to just close this PR and make a new one which switches people over